view_pdf_file/lib/main.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:advance_pdf_viewer/advance_pdf_viewer.dart';
+import 'package:flutter/material.dart';
+import 'package:view_pdf_file/constants.dart';
+import 'package:view_pdf_file/viewPDF.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+        title: 'View PDF File',
+        theme: ThemeData(
+          primarySwatch: Colors.blue,
+          visualDensity: VisualDensity.adaptivePlatformDensity,
+        ),
+        home: HomePage());
+  }
+}
+
+class HomePage extends StatefulWidget {
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  bool isLoading = false;
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
Split your build function into more logical and readable functions. This repo is for example apps that are for beginners. Too much nesting, makes it not readable.